app/controllers/users/settings_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Users
+  class SettingsController < ApplicationController
+    ALLOWED_PARAMS = %i[id
+                        config_theme
+                        config_font
+                        config_navbar
+                        display_announcements
+                        display_sponsors
+                        editor_version
+                        experience_level
+                        feed_fetched_at
+                        feed_mark_canonical
+                        feed_referential_link
+                        feed_url
+                        inbox_guidelines
+                        inbox_type
+                        permit_adjacent_sponsors
+                        user_id].freeze
+
+    def update
+      users_setting = Users::Setting.find(params[:id])
+      tab = params["users_setting"]["tab"] || "profile"
+
+      if users_setting.update(users_setting_params)
+        import_articles_from_feed(users_setting)
+
+        notice = "Your config has been updated. Refresh to see all changes."
```

Comment:
Again, I don't think this variable adds a lot but if you want to keep it, please move it closer to where it's actually used, it's a bit confusing up here (i.e. I was looking if it gets modified along the way).